packages/api-core/src/graphql/schema.graphql

```diff
@@ -34,6 +34,100 @@ enum MassUnit {
   oz
 }
 
+"Relational Operator Types used in filtering inside a single condition"
+enum RelationalOperatorTypes{
+  "Begins With used with String types"
+  beginsWith
+
+  "Ends With used with String types"
+  endsWith
+
+  "Equal"
+  eq
+
+  "Greater Than"
+  gt
+
+  "Greater Than or Equal"
+  gte
+
+  "In"
```

Comment:
These could still use more complete explanations of what they are. Not things like equals but not everyone knows what "Not In" means

Reply:
Added description for in, nin, regex, beginsWith, endWith